ci-operator/step-registry/azure/deprovision/resourcegroup/azure-deprovision-resourcegroup-ref.yaml

```diff
@@ -7,4 +7,4 @@ ref:
       cpu: 10m
       memory: 100Mi
   documentation: |-
-    Provision an empty resource group.
+    Deprovisions an empty resource group.
```

Comment:
Perhaps we can use a separate PR for the fix on this line, so that the other part of changes in this PR can fast merge ?

Reply:
I'll move a step under this folder per Jianlin's comment, so I'm keeping it this time. However, this is a good idea, and I'll take it into account in future PRs. Thank you @liangxia ^